serach.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+import json
+import milvus_model
+import sys, getopt
+from pymilvus import MilvusClient, DataType, Collection
+
+CLUSTER_ENDPOINT="http://localhost:27017"
+client = MilvusClient(uri=CLUSTER_ENDPOINT)
+
+client.load_collection("ceph_bkt_3cab575e_5080_4e5c_9675_86b8f53af5c7")
```

Comment:
how would the client know what is the collection name we created?
there should probably be a `kubectl` way to fetch it?

Reply:
Yaa we should expose it somehow